2.4/root/usr/libexec/fix-permissions

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 # Fix permissions on the given directory to allow group read/write of 
 # regular files and execute of directories.
-chgrp -R 0 $1
-chmod -R g+rw $1
+find $1 -exec chgrp -R 0 {} \;
```

Comment:
yep it is unnecessary

On Mon, Jun 13, 2016 at 10:10 AM, Vyacheslav Semushin <
notifications@github.com> wrote:

> In 2.4/root/usr/libexec/fix-permissions
> <https://github.com/sclorg/mongodb-container/pull/165#discussion_r66797574>
> :
>
> > @@ -1,6 +1,6 @@
> >  #!/bin/sh
> >  # Fix permissions on the given directory to allow group read/write of
> >  # regular files and execute of directories.
> > -chgrp -R 0 $1
> > -chmod -R g+rw $1
> > +find $1 -exec chgrp -R 0 {} \;
>
> Should we omit -R option here like we did for chmod?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sclorg/mongodb-container/pull/165/files/5042ad22c88cadfb5b455da52ed21e9e5b7bb86d#r66797574>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADbadDQ9B_VOZWq2qofKCSf6OaVBMqC9ks5qLWTAgaJpZM4I0XH7>
> .
>